common/src/main/java/gov/cms/ab2d/common/service/JobOutputServiceImpl.java

```diff
@@ -3,18 +3,18 @@
 import gov.cms.ab2d.common.model.Job;
 import gov.cms.ab2d.common.model.JobOutput;
 import gov.cms.ab2d.common.repository.JobOutputRepository;
+import lombok.AllArgsConstructor;
```

Comment:
The use of this trick (instead of explicitly spec'ing out the constructor, really makes a positive different in readability through the elimination of boilerplate.

Nothing to resolve here, just liked the technique.